### PR TITLE
Fetch page title when paste valid markdown title with one valid URL

### PR DESCRIPTION
**Pull request by abnersajr**
Now you can paste a valid markdown title with a link, eg: `# https://boostnote.io` and it gets the right page title.

boostnote

## Issue fixed

#2907 

## Type of changes

- :white_circle: Bug fix (Change that fixed an issue)
- :white_circle: Breaking change (Change that can cause existing functionality to change)
- :white_circle: Improvement (Change that improves the code. Maybe performance or development improvement)
- :radio_button: Feature (Change that adds new functionality)
- :white_circle: Documentation change (Change that modifies documentation. Maybe typo fixes)

## Checklist:

- :radio_button: My code follows [the project code style](docs/code_style.md)
- :radio_button: I have written test for my code and it has been tested
- :radio_button: All existing tests have been passed
- :radio_button: I have attached a screenshot/video to visualize my change if possible

### CODE REVIEW

File: browser/components/CodeEditor.js [hunk 1]
```diff
@@ -947,9 +948,11 @@ export default class CodeEditor extends React.Component {
     }
 
     const pastedTxt = clipboard.readText()
-
+    console.log(pastedTxt);
```
Comment (boniattirodrigo) [inline browser/components/CodeEditor.js]: I think you forgot to remove this line.

Comment (abnersajr) [reply browser/components/CodeEditor.js]: Thank you for your review. I've just fixed it.

**Comment by arcturus140** [comment]
does this prepend `##` to the URL?

**Comment by ZeroX-DG** [comment]
Can you fix the conflict?

**Comment by abnersajr** [comment]
@ZeroX-DG conflicts solved.

**Pull request merged**